TApplication has to be public for InitHelper to access it.
diff -ur tvision/include/tv/applictn.h tvision-py/include/tv/applictn.h
--- tvision/include/tv/applictn.h	2005-07-08 20:21:55.000000000 +0400
+++ tvision-py/include/tv/applictn.h	2018-02-16 19:09:52.000000000 +0300
@@ -17,7 +17,7 @@
 class CLY_EXPORT TApplication : public TProgram
 {
 
-protected:
+public:
 
     TApplication();
     virtual ~TApplication();
